pages/guide.md

```diff
@@ -99,3 +100,19 @@ Browsers that don't support HSTS are simply unaffected by HSTS, so there is no h
 [Pulse](https://pulse.cio.gov) looks for server-side redirects, using an appropriate HTTP response code. Sites that use client-side redirects -- such as a &lt;meta refresh&gt; tag or JavaScript -- will not be seen as redirects. To meet the M-15-13 requirement of enforcing HTTPS, agencies should employ server-side redirects (or alternatively, disable HTTP access altogether).
 
 Sites that are reachable on both a root domain (`http://agency.gov`) and their www subdomain (`http://www.agency.gov`) should perform a redirect to HTTPS in both cases. Redirecting one but not the other could also cause Pulse to indicate that a domain does not enforce HTTPS.
+
+### Are federally operated certificate revocation services (CRL, OCSP) also required to move to HTTPS?
+
+No. This very narrow class of services, that provide CRL and OCSP information for the purposes of verifying the revocation status of certificates used to make other HTTPS connections, should abide by industry best practices and their respective specifications.
+
+For CRL, [RFC 5280](https://tools.ietf.org/html/rfc5280) says:
+
+```
+CAs SHOULD NOT include URIs that specify https, ldaps, or similar schemes in extensions.  CAs that include an https URI in one of these extensions MUST ensure that the server's certificate can be validated without using the information that is pointed to by the URI.  Relying parties that choose to validate the server's certificate when obtaining information pointed to by an https URI in the cRLDistributionPoints, authorityInfoAccess, or subjectInfoAccess extensions MUST be prepared for the possibility that this will result in unbounded recursion.
```

Comment:
That's reasonable, I wasn't thinking about mobile. Computers.

On Thu, Mar 24, 2016 at 5:57 PM, Eric Mill <notifications@github.com> wrote:

> In pages/guide.md
> <https://github.com/GSA/https/pull/187#discussion_r57394511>:
>
> > @@ -99,3 +100,19 @@ Browsers that don't support HSTS are simply unaffected by HSTS, so there is no h
> >  [Pulse](https://pulse.cio.gov) looks for server-side redirects, using an appropriate HTTP response code. Sites that use client-side redirects -- such as a &lt;meta refresh&gt; tag or JavaScript -- will not be seen as redirects. To meet the M-15-13 requirement of enforcing HTTPS, agencies should employ server-side redirects (or alternatively, disable HTTP access altogether).
> >
> >  Sites that are reachable on both a root domain (`http://agency.gov`) and their www subdomain (`http://www.agency.gov`) should perform a redirect to HTTPS in both cases. Redirecting one but not the other could also cause Pulse to indicate that a domain does not enforce HTTPS.
> > +
> > +### Are federally operated certificate revocation services (CRL, OCSP) also required to move to HTTPS?
> > +
> > +No. This very narrow class of services, that provide CRL and OCSP information for the purposes of verifying the revocation status of certificates used to make other HTTPS connections, should abide by industry best practices and their respective specifications.
> > +
> > +For CRL, [RFC 5280](https://tools.ietf.org/html/rfc5280) says:
> > +
> > +```
> > +CAs SHOULD NOT include URIs that specify https, ldaps, or similar schemes in extensions.  CAs that include an https URI in one of these extensions MUST ensure that the server's certificate can be validated without using the information that is pointed to by the URI.  Relying parties that choose to validate the server's certificate when obtaining information pointed to by an https URI in the cRLDistributionPoints, authorityInfoAccess, or subjectInfoAccess extensions MUST be prepared for the possibility that this will result in unbounded recursion.
>
> Fair, but I'm optimizing for the website! Which I'd like to look good in
> mobile, desktop, and other form factors...
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/GSA/https/pull/187/files/da22f9f580c5f2360f680da0f4cdbad9be2601ab#r57394511>
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: D1B3 ADC0 E023 8CA6